pkg/api/vwo.go

```diff
@@ -91,3 +91,11 @@ func WithDevelopmentMode() VWOOption {
 		vwo.IsDevelopmentMode = true
 	}
 }
+
+// WithGoalAttributes sets GoalTypeToTrack to the passed type and ShouldTrackReturningUser to false
+func WithGoalAttributes(goalTypeToTrack interface{}, shouldTrackReturningUser interface{}) VWOOption {
```

Comment:
can someone please let me know what will be the code of the launch API usage if only shouldTrackReturningUser needs to be set to true?

Reply:
As stated in https://github.com/decabits/vwo-golang-example-app/blob/feat/track/models/vwo.go file on line 26/23.
instance, err := vwo.Launch(settingsFile, api.WithGoalAttributes(nil, true)) 
We can call Launch like this, without Development mode, Custom logger or User storage, just passing shouldTrackReturningUser's value as true.